tests/updates/base_test.py

```diff
@@ -6,10 +6,13 @@
 from git_theta import utils
 
 
+ENV_UPDATE_TYPE = "GIT_THETA_UPDATE_TYPE"
```

Comment:
I there a reason for duplicating the string over using `utils.EnvVarConstants.UPDATE_TYPE.name?

Reply:
So with that `__get__` method defined, any time you access a member of EnvVarConstants it automatically turns the accessed result into whatever `__get__` returns. So `utils.EnvVarConstants.UPDATE_TYPE` automatically turns into a string like `"dense"` so we lose the `.name` attribute.

Reply:
Makes sense